src/vivarium_gates_mncnh/validation/formatting.py

```diff
@@ -0,0 +1,30 @@
+import pandas as pd
+from vivarium_testing_utils.automated_validation.constants import DRAW_INDEX, SEED_INDEX
+from vivarium_testing_utils.automated_validation.data_transformation import calculations
+from vivarium_testing_utils.automated_validation.data_transformation.formatting import (
+    TotalPopulationPersonTime,
+)
+
+
+class TotalLiveBirths(TotalPopulationPersonTime):
+    """Formatter for simulation data that contains total live births."""
+
+    def __init__(self, scenario_columns: list[str]) -> None:
```

Comment:
I can't call super because there are hard coded values for measure and file names I can't work around.

Reply:
You can call super, and then reassign the attributes that have been set incorrectly. If that doesn't save you much, it may not be worth it though.